tensorflow/.buildspec_templates/std_buildspec_template.yml

```diff
@@ -0,0 +1,94 @@
+  account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+  region: &REGION <set-$REGION-in-environment>
+  framework: &FRAMEWORK tensorflow
+  version: &VERSION 1.2.3
+  short_version: &SHORT_VERSION 1.2
+  cuda_version: &CUDA_VERSION cu102
```

Comment:
Doesn't the cuda version and os version come under GPU/CPU Image sections?

Reply:
No. That is a change we intend to make in the future, but haven't made on this repo yet through a PR.

Thanks for pointing it out.

Reply:
let's add that variable now itself so that it wont' break when this feature is added in the future?